decentraland/components/presentational/MyAccount.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Colors from "@constants/Colors";
+import PropTypes from "prop-types";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+import MyAccountCreationStatusItem from "./MyAccountCreationStatusItem";
+import ActionStatus from "@constants/ActionStatus";
+import MyAccountProgress from "./MyAccountProgress";
+
+export default function MyAccount({ progress, creationActions }) {
+  return (
+    <View style={styles.container}>
+      <MyAccountProgress progress={progress} />
+      <View style={styles.actionItemsContainer}>
+        {creationActions.map(action => {
+          return (
+            <MyAccountCreationStatusItem
+              key={action.name}
```

Comment:
Hey @mpetrunic, as code style we are using destructuring assign whenever is possible.